src/raster/rasterband.rs

```diff
@@ -431,12 +431,27 @@ impl<'a> RasterBand<'a> {
     }
 
     /// Set the no data value of this band.
+    ///
+    /// See [`set_no_data`](Self::set_no_data) for similar function
+    /// where _no-data_ can also be deleted.
     pub fn set_no_data_value(&mut self, no_data: f64) -> Result<()> {
-        let rv = unsafe { gdal_sys::GDALSetRasterNoDataValue(self.c_rasterband, no_data) };
+        self.set_no_data(Some(no_data))
+    }
```

Comment:
If backward compatibility weren't an option, I would have changed this signature of this to accept `Option<f64>` over creating a new method and delegating to it. Should we consider a breaking change here?